lesson09.md

```diff
@@ -0,0 +1,216 @@
+    return degreesInFahrenheit;
+}
+```
+
+---
+
+Create a function that takes a day of the week and returns a message depending on the day:
+
+```js
+function messageOfTheDay(day) {
+    if (day === "Monday") {
+        return "Sucks";
+    }
+    if (day === "Friday") {
+        return "Almost Weekend!";
+    }
+    if (day === "Saturday" || day === "Sunday") {
+        return "Yippee!!!";
+    }
+    return "It's OK.";
+}
+```
+
+---
+
+### Variable lifetime
+
+```js
+let greeting = "Hello";
+
+if (greeting === "Hello") {
+    let name = "Sevtap";
+    console.log(greeting + " " + name);s
+}
+```
+
+What's the lifetime of `greeting` and `name`?
+
+---
+
+* Variables defined inside a code block `{}` exist until the end of the code block
+* Variables defined in the global scope (e.g. outside of code blocks) exist until the user closes the page.
+
+---
+
+### HTML and JavaScript
+
+We've learned that we can load a JavaScript file from within HTML:
+
+```html
+<script src="main.js"></script>
+```
+
+And we can use `console.log` to output values to the debug console.
+
+---
+
+Let's try to actually change something in our web page. The browser defined a lot of variables for us to play with, e.g.:
+
+```js
+// set the document's background color by assigning
+// a color to this variable:
+document.body.style.backgroundColor = "red";
+```
+
+You can choose your favorite color from [https://developer.mozilla.org/en-US/docs/Web/CSS/color_value](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value)
+
+---
+
+We can also change the font size, e.g.:
+
+```js
+document.body.style.fontSize = "1.5em";
+```
+
+(`em` is a unit in typography, see [https://developer.mozilla.org/en-US/docs/Web/CSS/font-size#ems](https://developer.mozilla.org/en-US/docs/Web/CSS/font-size#ems))
+
+---
+
+So now we know how to set background color and font size.
+
+All the code that we write in our `main.js` executes when the page loads.
+
+But what if we would like to run some JavaScript code when the user interacts with the HTML page, e.g. clicks a button?
+
+---
+
+We can use `onclick` event:
+
+```html
+<button onclick="console.log('hello');">Click me!</button>
+```
+
+We can write any JavaScript code in the `onclick` HTML attribute!
+
+---
+
+But do we really want to write all the JavaScript code inside a `onclick` HTML attribute?
+
+What could we use instead?
+
+Functions!<!-- .element: class="fragment" -->
+
+---
+
+Try it!s
+
+HTML:
+```html
+<button onclick="mySuperFunction()">Click me!</button>
+```
+
+JavaScript:
+```js
+function mySuperFunction() {
+    console.log("My button was clicked!!!!");
+}
+```
+
+Bonus: Can you output how often your button as clicked?
+
+---
+
+Solution:
+
+```js
+let clickCount = 0;
+function mySuperFunction() {
+    console.log("Number of clicks:", clickCount);
+}
+```
+
+---
+
+Let's practice!
+
+* Add a button and when the user clicks the button, set the background color to `lightblue`
```

Comment:
I think the exercise in the class could be only changing the color to green/blue,
and making it darker could be a homework